pyproject.toml

```diff
@@ -1,6 +1,6 @@
 [tool.poetry]
 name = "serverless-builder"
-version = "1.0.1"
```

Comment:
We change `version` only during release, and release happens when you create new tag. Release itself should happen after PR is merged :) Please revert.

Reply:
hint: bump2version